gdal/gdaladdo.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="gdaladdo" name="GDAL addo" version="@VERSION@">
```

Comment:
gdal_addo

Reply:
I understand what you want here.
But some tools are called like `gdaladdo -p "xxx" ..` and other  `gdal_merge -p "xxx" ..`
I can establish something like 'gdal-gdaladdo', 'gdal-gdal_merge' and 'gdal-ogrinfo' ...

Reply:
Ah, I see. `gdal-gdaladdo` would make sense to me.